linux/LinuxMDEparser/README.ME

```diff
@@ -0,0 +1,23 @@
+# LinuxMDEparser
```

Comment:
Looks like it requires Python modules that must be installed separately. Please describe how to. (pip install XlsxWriter? anything else?) 

You can also consider adding instructions for building an .exe file as a separate section to this README, if you believe it will be helpful to users

Reply:
I'll update and resubmit README.MD

Reply:
We do not have releases in this repository right now, as it is created under microsoft organization, and it was in the repo spec when it was created. You can build and host releases outside of this repo (in your repo?) and add a link to the releases here. Will it work?